Description: Shorten test initialization timeouts
 Shorten timeouts in tests for virtuoso initialization to 60 seconds. This
 causes resources tie-ups on buildds where a FTBFS result in a silently
 binary.
Author: Obey Arthur Liu <arthur@milliways.fr>
Last-Update: 2010-05-12
--- a/appsrc/ODS-Addressbook/make_vad.sh
+++ b/appsrc/ODS-Addressbook/make_vad.sh
@@ -93,7 +93,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/appsrc/ODS-Blog/make_vad.sh
+++ b/appsrc/ODS-Blog/make_vad.sh
@@ -112,7 +112,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
     then
--- a/appsrc/ODS-Bookmark/make_vad.sh
+++ b/appsrc/ODS-Bookmark/make_vad.sh
@@ -97,7 +97,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/appsrc/ODS-Briefcase/make_vad.sh
+++ b/appsrc/ODS-Briefcase/make_vad.sh
@@ -113,7 +113,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/appsrc/ODS-Calendar/make_vad.sh
+++ b/appsrc/ODS-Calendar/make_vad.sh
@@ -96,7 +96,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/appsrc/ODS-Community/make_vad.sh
+++ b/appsrc/ODS-Community/make_vad.sh
@@ -100,7 +100,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/appsrc/ODS-Discussion/make_vad.sh
+++ b/appsrc/ODS-Discussion/make_vad.sh
@@ -110,7 +110,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
 	then
--- a/appsrc/ODS-FeedManager/make_vad.sh
+++ b/appsrc/ODS-FeedManager/make_vad.sh
@@ -96,7 +96,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/appsrc/ODS-Framework/make_vad.sh
+++ b/appsrc/ODS-Framework/make_vad.sh
@@ -138,7 +138,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ]
   then
--- a/appsrc/ODS-Framework/oauth/make_vad.sh
+++ b/appsrc/ODS-Framework/oauth/make_vad.sh
@@ -188,7 +188,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/appsrc/ODS-Gallery/make_vad.sh
+++ b/appsrc/ODS-Gallery/make_vad.sh
@@ -101,7 +101,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/appsrc/ODS-Polls/make_vad.sh
+++ b/appsrc/ODS-Polls/make_vad.sh
@@ -95,7 +95,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/appsrc/ODS-WebMail/make_vad.sh
+++ b/appsrc/ODS-WebMail/make_vad.sh
@@ -96,7 +96,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/appsrc/ODS-Wiki/make_vad.sh
+++ b/appsrc/ODS-Wiki/make_vad.sh
@@ -131,7 +131,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/binsrc/b3s/make_vad.sh
+++ b/binsrc/b3s/make_vad.sh
@@ -203,7 +203,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ]
   then
--- a/binsrc/bpel/make_vad.sh
+++ b/binsrc/bpel/make_vad.sh
@@ -254,7 +254,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
 	then
--- a/binsrc/hosting/mono/tests/tclrsrv.sh
+++ b/binsrc/hosting/mono/tests/tclrsrv.sh
@@ -217,7 +217,7 @@
       ddate=`date`
       starth=`date | cut -f 2 -d :`
       starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-      timeout=600
+      timeout=60
       rm -f *.lck
       $SERVER +foreground -c tclr.ini $* 1>/dev/null & 
       stat="true"
--- a/binsrc/isparql/make_vad.sh
+++ b/binsrc/isparql/make_vad.sh
@@ -117,7 +117,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ]
     then
--- a/binsrc/rdf_mappers/make_vad.sh
+++ b/binsrc/rdf_mappers/make_vad.sh
@@ -234,7 +234,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/binsrc/samples/demo/make_vad.sh
+++ b/binsrc/samples/demo/make_vad.sh
@@ -115,7 +115,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/binsrc/samples/demo/mkdemo.sh
+++ b/binsrc/samples/demo/mkdemo.sh
@@ -140,7 +140,7 @@
 
 START_SERVER()
 {
-  timeout=180
+  timeout=60
 
   ECHO "Starting Virtuoso DEMO server ..."
   if [ "z$HOST_OS" != "z" ] 
--- a/binsrc/samples/demo/mkdoc.sh
+++ b/binsrc/samples/demo/mkdoc.sh
@@ -161,7 +161,7 @@
 {
    if [ "z$DEMODB" = "z" ]
    then
-       timeout=180
+       timeout=60
 
        ECHO "Starting Virtuoso server ..."
        if [ "z$HOST_OS" != "z" ] 
--- a/binsrc/samples/sparql_demo/make_vad.sh
+++ b/binsrc/samples/sparql_demo/make_vad.sh
@@ -113,7 +113,7 @@
     ddate=`date`
     starth=`date | cut -f 2 -d :`
     starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-    timeout=600
+    timeout=60
     $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
     then
--- a/binsrc/sync/make_vad.sh
+++ b/binsrc/sync/make_vad.sh
@@ -104,7 +104,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "x$HOST_OS" != "x" ]
   then
--- a/binsrc/tutorial/make_vad.sh
+++ b/binsrc/tutorial/make_vad.sh
@@ -104,7 +104,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "z$HOST_OS" != "z" ] 
   then
--- a/binsrc/vsp/admin/debug/make_vad.sh
+++ b/binsrc/vsp/admin/debug/make_vad.sh
@@ -70,7 +70,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   $myrm -f *.lck
   if [ "x$HOST_OS" != "x" ]
 	then
--- a/binsrc/vspx/suite/vspx_suite.sh
+++ b/binsrc/vspx/suite/vspx_suite.sh
@@ -44,7 +44,7 @@
   ddate=`date`
   starth=`date | cut -f 2 -d :`
   starts=`date | cut -f 3 -d :|cut -f 1 -d " "`
-  timeout=600
+  timeout=60
   rm -f *.lck
   $SERVER
   stat="true"
--- a/binsrc/yacutia/mkvad.sh
+++ b/binsrc/yacutia/mkvad.sh
@@ -148,7 +148,7 @@
 
 START_SERVER()
 {
-  timeout=120
+  timeout=60
 
   ECHO "Starting Virtuoso server ..."
   if [ "z$HOST_OS" != "z" ]
--- a/binsrc/bpel/tests/tpcc/run.sh
+++ b/binsrc/bpel/tests/tpcc/run.sh
@@ -167,17 +167,17 @@
 cat ../dbservices.wsdl | sed -e "s/HTTPPORTDB/$HTTPPORTDB/g" > bpeltpcc/dbservices.wsdl
 cat ../tdservices.wsdl | sed -e "s/HTTPPORTTD/$HTTPPORTTD/g" > bpeltpcc/tdservices.wsdl
 MakeConfig $DS1 $HTTPPORTBP
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cd ..
 
 cd db
 MakeConfig $DS2 $HTTPPORTDB
-START_SERVER $DS2 1000
+START_SERVER $DS2 60
 cd ..
 
 cd td
 MakeConfig $DS3 $HTTPPORTTD
-START_SERVER $DS3 1000
+START_SERVER $DS3 60
 cd ..
 
 # Configure endpoint hosts
--- a/binsrc/tests/biftest/tests/tjavavm.sh
+++ b/binsrc/tests/biftest/tests/tjavavm.sh
@@ -103,7 +103,7 @@
 MAKECFG_FILE ../../suite/$TESTCFGFILE $PORT $CFGFILE
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < java_ts.sql
 
--- a/binsrc/tests/biftest/thook.sh
+++ b/binsrc/tests/biftest/thook.sh
@@ -23,7 +23,7 @@
 
 OUTPUT=thook.output
 ISQL=../isql
-TIMEOUT=1000
+TIMEOUT=60
 HOST_OS=`uname -s | grep WIN`
 SERVER=./virtuoso-iodbc-sample-t
 PORT=${PORT-1111}
@@ -169,7 +169,7 @@
 echo $ISQL $DSN dba dba PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT "EXEC=raw_exit()" >> $OUTPUT
 $ISQL $DSN dba dba PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT "EXEC=raw_exit()" >> $OUTPUT
 rm -rf virtuoso.db virtuoso.log virtuoso.lck virtuoso.trx core
-START_SERVER $PORT 1000
+START_SERVER $PORT $TIMEOUT
 RUN $ISQL $DSN dba dba PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT thook.sql
 
 RUN $ISQL $DSN MANAGER MANAGER PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT thook1.sql
--- a/binsrc/tests/dotnet/test_server.sh
+++ b/binsrc/tests/dotnet/test_server.sh
@@ -149,7 +149,7 @@
 
 if [ -f VirtuosoClientSuite/OpenLink.Data.$CLIENT ] ; then
 
-START_SERVER $PORT 10000
+START_SERVER $PORT 60
 
 cd VirtuosoClientSuite
 if [ "x$DTC_TEST" != "x" ]
--- a/binsrc/tests/lubm/tlubm.sh
+++ b/binsrc/tests/lubm/tlubm.sh
@@ -40,7 +40,7 @@
 fi
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 BANNER "LUBM LOAD"
 
--- a/binsrc/tests/suite/bpel.sh
+++ b/binsrc/tests/suite/bpel.sh
@@ -225,7 +225,7 @@
 if [ "$TEST_ECHO" = "yes" ] ; then
 
 CNT=`expr $CNT + 1`
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 
 LOG "ECHO BPEL script test"
 DoCommand $DS1 "vad_install ('bpel_filesystem.vad', 0, 1);" "bpel_filesystem.vad install"
@@ -260,7 +260,7 @@
 DSN=$DS1
 SHUTDOWN_SERVER
 CNT=`expr $CNT + 1`
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cp ../bpel_filesystem.vad ./
 DoCommand $DS1 "vad_install ('bpel_filesystem.vad', 0, 1);" "bpel_filesystem.vad install"
 #DoCommand $DS1 "trace_on('soap');" "TRACE ON"
@@ -336,7 +336,7 @@
 MakeConfig $DS2 $HP2
 DSN=$DS2
 SHUTDOWN_SERVER
-START_SERVER $DS2 1000
+START_SERVER $DS2 60
 RUN $ISQL $DS2 PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < ../fault1/ini2.sql
 #DoCommand $DS2 "vhost_define (lpath=>'/BPELREQ/', ppath=>'/SOAP/', soap_user=>'DBA');" "Request vhost"
 cd ../fault1
@@ -345,7 +345,7 @@
 DSN=$DS1
 SHUTDOWN_SERVER
 CNT=`expr $CNT + 1`
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cp ../bpel_filesystem.vad ./
 DoCommand $DS1 "vad_install ('bpel_filesystem.vad', 0, 1);" "bpel_filesystem.vad install"
 DoCommand $DS1 "create user BPELTEST;" "create user BPELTEST"
@@ -356,13 +356,13 @@
 RUN $ISQL $DS1 BPELTEST BPELTEST PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT -u 'http_port_two=$HP2' < ini.sql
 SHUTDOWN_SERVER
 CNT=`expr $CNT + 1`
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 RUN $ISQL $DS1 BPELTEST BPELTEST PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT -u 'http_port_two=$HP2' < check.sql
 DSN=$DS2
 SHUTDOWN_SERVER
 RUN $ISQL $DS1 BPELTEST BPELTEST PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT -u 'http_port_two=$HP2' < inv.sql
 cd ../fault1_req
-START_SERVER $DS2 1000
+START_SERVER $DS2 60
 cd ../fault1
 DSN=$DS1
 RUN $ISQL $DS1 dba dba PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT -u 'http_port_two=$HP2' < processXQuery/ini.sql
--- a/binsrc/tests/suite/byteorder.sh
+++ b/binsrc/tests/suite/byteorder.sh
@@ -87,7 +87,7 @@
 
 SHUTDOWN_SERVER
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < byteorder.sql
 
--- a/binsrc/tests/suite/gtkbench.sh
+++ b/binsrc/tests/suite/gtkbench.sh
@@ -70,7 +70,7 @@
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 
 LOG "Creating the schema"
 RUN $GTKBENCH $LOGIN -C
--- a/binsrc/tests/suite/inprocess.sh
+++ b/binsrc/tests/suite/inprocess.sh
@@ -36,7 +36,7 @@
 rm -f $DBLOGFILE
 rm -f $DBFILE
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $INS $DSN 1000 100 dba dba usedt
 if test $STATUS -ne 0
--- a/binsrc/tests/suite/large_db.sh
+++ b/binsrc/tests/suite/large_db.sh
@@ -44,7 +44,7 @@
 then
 
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < large_db_5g.sql
 
@@ -69,7 +69,7 @@
 LOG "Staring server with  $OBACKUP_REP_OPTION large_"
 RUN $SERVER $FOREGROUND_OPTION $OBACKUP_REP_OPTION "large_"
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < large_db_check.sql
 
 SHUTDOWN_SERVER
@@ -104,7 +104,7 @@
 
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < large_db_3g.sql
 
@@ -126,7 +126,7 @@
 LOG "Staring server with  $OBACKUP_REP_OPTION large_"
 RUN $SERVER $FOREGROUND_OPTION $OBACKUP_REP_OPTION "large_"
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < large_db_3g_check.sql
 
 SHUTDOWN_SERVER
@@ -162,7 +162,7 @@
 
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < large_db_5g.sql
 
--- a/binsrc/tests/suite/msdtc.sh
+++ b/binsrc/tests/suite/msdtc.sh
@@ -146,7 +146,7 @@
 
 LOGFILE=../msdtc.output
 cp -r ../plugins ./
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cd ..
 
 rm -rf msdtc2
@@ -156,7 +156,7 @@
 mkINI "virtuoso.ini" $DS2
 
 cp -r ../plugins ./
-START_SERVER $DS2 1000
+START_SERVER $DS2 60
 cd ..
 
 LOGFILE=msdtc.output
@@ -218,7 +218,7 @@
 sleep 10
 
 
-START_SERVER $DS2 1000
+START_SERVER $DS2 60
 cd ..
 
 RUN $ISQL $DS2 PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < msdtc_conn_check.sql
@@ -244,7 +244,7 @@
 
 LOGFILE=../msdtc.output
 cp -r ../plugins ./
-START_SERVER $DS3 1000
+START_SERVER $DS3 60
 cd ..
 
 RUN $ISQL $DS2 PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < msdtc_conn_check.sql
--- a/binsrc/tests/suite/nwxml.sh
+++ b/binsrc/tests/suite/nwxml.sh
@@ -35,7 +35,7 @@
 cp "${HOME}/binsrc/samples/demo/noise.txt" .
 rm -f $DELETEMASK
 MAKECFG_FILE_WITH_HTTP $TESTCFGFILE $PORT $HTTPPORT $CFGFILE
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tlogft1.sql
 if test $STATUS -ne 0
@@ -45,7 +45,7 @@
 fi
 
 STOP_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tlogft2.sql
 if test $STATUS -ne 0
@@ -98,7 +98,7 @@
 #fi
 
 STOP_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 #RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < nwxml3b.sql
 #if test $STATUS -ne 0
@@ -151,7 +151,7 @@
 fi
 
 STOP_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 #RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < nwxmltype3b.sql
 #if test $STATUS -ne 0
--- a/binsrc/tests/suite/obackup.sh
+++ b/binsrc/tests/suite/obackup.sh
@@ -46,7 +46,7 @@
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000 $NO_CP_OPT
+START_SERVER $PORT 60 $NO_CP_OPT
 
 RUN $INS $DSN 100000  100
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < nwdemo_norefs.sql
@@ -55,7 +55,7 @@
 
 RUN $ISQL $DSN '"EXEC=shutdown();"' ERRORS=STDOUT
 
-START_SERVER $PORT 1000 $NO_CP_OPT
+START_SERVER $PORT 60 $NO_CP_OPT
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < nwdemo_update.sql
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < obackup.sql
@@ -69,7 +69,7 @@
 echo "Staring server with  $OBACKUP_REP_OPTION nwdemo_i_#"
 RUN $SERVER $FOREGROUND_OPTION $OBACKUP_REP_OPTION "nwdemo_i_#" $OBACKUP_DIRS_OPTION nw1,nw2,nw3,nw4,nw5
 
-START_SERVER $PORT 1000 $NO_CP_OPT
+START_SERVER $PORT 60 $NO_CP_OPT
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < obackup1.sql
 
@@ -83,7 +83,7 @@
 echo "Staring server with  $OBACKUP_REP_OPTION nwdemo_i_#"
 RUN $SERVER $FOREGROUND_OPTION $OBACKUP_REP_OPTION "nwdemo_i_#" $OBACKUP_DIRS_OPTION nw1,nw2,nw3,nw4,nw5
 
-START_SERVER $PORT 1000 $NO_CP_OPT
+START_SERVER $PORT 60 $NO_CP_OPT
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < obackupck.sql
 
@@ -102,7 +102,7 @@
 echo "Staring server with  $OBACKUP_REP_OPTION vvv"
 RUN $SERVER $FOREGROUND_OPTION $OBACKUP_REP_OPTION "vvv"
 
-START_SERVER $PORT 1000 $NO_CP_OPT
+START_SERVER $PORT 60 $NO_CP_OPT
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < obackup_userck2.sql
 
@@ -127,7 +127,7 @@
 
 RUN $ISQL $DSN '"EXEC=shutdown();"' ERRORS=STDOUT
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 echo "Updating tpcc table, stage 1"
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tpcc_update1.sql
@@ -139,7 +139,7 @@
 echo "Staring server with  $OBACKUP_REP_OPTION tpcc_k_#"
 RUN $SERVER $FOREGROUND_OPTION $OBACKUP_REP_OPTION "tpcc_k_#"
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 echo "Checking tpcc database"
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < ob_tpcc_check.sql
@@ -154,7 +154,7 @@
 echo "Staring server with  $OBACKUP_REP_OPTION tpcc_i_#"
 RUN $SERVER $FOREGROUND_OPTION $OBACKUP_REP_OPTION "tpcc_i_#"
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 echo "Checking tpcc database..."
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < ob_tpcc_check.sql
--- a/binsrc/tests/suite/oledb.sh
+++ b/binsrc/tests/suite/oledb.sh
@@ -116,7 +116,7 @@
 
 #REGISTER_PROVIDER
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < nwdemo.sql
 if [ $STATUS -ne 0 ]; then
--- a/binsrc/tests/suite/rtest.sh
+++ b/binsrc/tests/suite/rtest.sh
@@ -65,7 +65,7 @@
 cd remote1
 MAKECFG_FILE ../$TESTCFGFILE $DS1 $CFGFILE
 ln -s ../words.esp words.esp
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cd ..
 
 rm -rf remote2
@@ -73,7 +73,7 @@
 cd remote2
 MAKECFG_FILE ../$TESTCFGFILE $DS2 $CFGFILE
 ln -s ../words.esp words.esp
-START_SERVER $DS2 1000
+START_SERVER $DS2 60
 cd ..
 
 LOG "Loading base tables"
@@ -162,7 +162,7 @@
 fi
 
 cd remote1
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cd ..
 
 RUN $ISQL $DS2 PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < rtest5.sql
--- a/binsrc/tests/suite/tblob_recode.sh
+++ b/binsrc/tests/suite/tblob_recode.sh
@@ -86,7 +86,7 @@
 
 SHUTDOWN_SERVER
 rm -f $DBLOGFILE $DBFILE
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 
 gunzip -c pdd_txt.gz >pdd.txt
 
--- a/binsrc/tests/suite/tclstart.sh
+++ b/binsrc/tests/suite/tclstart.sh
@@ -52,4 +52,4 @@
 export BACKUP_DUMP_OPTION CRASH_DUMP_OPTION TESTCFGFILE SERVER
 
 . ./test_fn.sh
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
--- a/binsrc/tests/suite/tcpt.sh
+++ b/binsrc/tests/suite/tcpt.sh
@@ -37,7 +37,7 @@
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
 STOP_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $INS $DSN 100000 1
 
@@ -45,7 +45,7 @@
 
 rm -f $LOCKFILE
 #exit;
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tcptck.sql
 
@@ -60,14 +60,14 @@
 rm -f $DBFILE
 rm -f *.cpt-after-recov *.trx-after-recov *.cpt
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $INS $DSN 100000 1
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT -u "FLAG=2" < tcptdt.sql
 
 rm -f $LOCKFILE
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tcptck.sql
 
--- a/binsrc/tests/suite/tdav.sh
+++ b/binsrc/tests/suite/tdav.sh
@@ -81,7 +81,7 @@
 _dsn=$DSN
 DSN=$DS1
 SHUTDOWN_SERVER
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 CHECK_HTTP_PORT $HTTPPORT
 
 
--- a/binsrc/tests/suite/tdav_meta.sh
+++ b/binsrc/tests/suite/tdav_meta.sh
@@ -89,7 +89,7 @@
 _dsn=$DSN
 DSN=$DS1
 SHUTDOWN_SERVER
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 CHECK_HTTP_PORT $HTTPPORT
 
 LOG "test1"
--- a/binsrc/tests/suite/tdbp.sh
+++ b/binsrc/tests/suite/tdbp.sh
@@ -183,7 +183,7 @@
 
 SHUTDOWN_SERVER
 rm -f $DBLOGFILE $DBFILE
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 
 gunzip -c pdd_txt.gz >pdd.txt
 
@@ -234,7 +234,7 @@
 LOG "Removing database"
 rm -f $DBLOGFILE $DBFILE
 LOG "Starting an empty database"
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 
 
 GenPUMP2 $SAVDSN t2.args
@@ -245,7 +245,7 @@
 DoCommand  $SAVDSN 'select U_NAME, U_PASSWORD, U_GROUP, U_ID, U_DATA, U_IS_ROLE, U_SQL_ENABLE from "DB"."DBA"."SYS_USERS";' "Getting users"
 
 STOP_SERVER
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tmulgrp2.sql
 if test $STATUS -ne 0
--- a/binsrc/tests/suite/tdconv.sh
+++ b/binsrc/tests/suite/tdconv.sh
@@ -210,7 +210,7 @@
 
 SHUTDOWN_SERVER
 rm -f $DBLOGFILE $DBFILE
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 
 
 
--- a/binsrc/tests/suite/tdrop.sh
+++ b/binsrc/tests/suite/tdrop.sh
@@ -73,7 +73,7 @@
 rm -f $DBFILE
 MakeIni
 
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 
 
 LOG "FILLING TPC-C DB"
@@ -86,7 +86,7 @@
 ../tpcc "localhost:$DS1" $USR $PWD i 1 
 
 SHUTDOWN_SERVER
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 
 #RUN $ISQL $DS1 '"EXEC=load tdrop1.sql;"' ERRORS=stdout >> $LOGFILE 
 
@@ -108,13 +108,13 @@
 
 SHUTDOWN_SERVER
 
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 RUN $ISQL $DS1 PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tdrop1.sql
 # Kill the server with raw_exit
 RUN $ISQL $DS1 '"EXEC=raw_exit();"' ERRORS=stdout
 
 # check dropped table
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 RUN $ISQL $DS1 PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tdrop2.sql
 
 SHUTDOWN_SERVER
--- a/binsrc/tests/suite/testall.sh
+++ b/binsrc/tests/suite/testall.sh
@@ -150,7 +150,7 @@
 #
 #  Start the server once:
 #
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 
 
@@ -273,7 +273,7 @@
 #
 rm -f $DELETEMASK
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 LOG "Getting the status of database into file ident.txt"
 if $ISQL $DSN "EXEC=status();" VERBOSE=OFF ERRORS=STDOUT > ident.txt
--- a/binsrc/tests/suite/thttp.sh
+++ b/binsrc/tests/suite/thttp.sh
@@ -799,7 +799,7 @@
      BLOG_TEST=0  
      LOG "ODS & Blog2 VAD packages are not built"
    fi
-   START_SERVER $PORT 1000
+   START_SERVER $PORT 60
    sleep 1
    if [ $BLOG_TEST -eq 1 ]
    then
--- a/binsrc/tests/suite/timsg.sh
+++ b/binsrc/tests/suite/timsg.sh
@@ -73,7 +73,7 @@
    ;;
 esac   
 
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 
 #make another test file, the original text_1947.db.test is no longer in the db
 cp words.esp test_1947.db.test
--- a/binsrc/tests/suite/tjdbc.sh
+++ b/binsrc/tests/suite/tjdbc.sh
@@ -41,7 +41,7 @@
 rm -f $DBFILE
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 ECHO "STARTED: JDBC 2 Test suite"
 cd $JDBCDIR
@@ -81,7 +81,7 @@
 rm -f $DBFILE
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 ECHO "STARTED: JDBC 3 Test suite"
 cd $JDBCDIR
--- a/binsrc/tests/suite/tpcd.sh
+++ b/binsrc/tests/suite/tpcd.sh
@@ -58,7 +58,7 @@
 mkdir tpcdremote1
 cd tpcdremote1
 MAKECFG_FILE ../$TESTCFGFILE $DS1 $CFGFILE
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cd ..
 
 LOG
@@ -91,7 +91,7 @@
 mkdir tpcdremote2
 cd tpcdremote2
 MAKECFG_FILE ../$TESTCFGFILE $DS2 $CFGFILE
-START_SERVER $DS2 1000
+START_SERVER $DS2 60
 cd ..
 
 LOG
--- a/binsrc/tests/suite/tproviders.sh
+++ b/binsrc/tests/suite/tproviders.sh
@@ -51,7 +51,7 @@
 NIGHTLY_PORT=$PORT
 export NIGHTLY_PORT
 
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 
 #
 #   Run Jena tests
--- a/binsrc/tests/suite/tproxy.sh
+++ b/binsrc/tests/suite/tproxy.sh
@@ -243,7 +243,7 @@
    GenHTML
    GenVSP
    STOP_SERVER
-   START_SERVER $DSN 1000
+   START_SERVER $DSN 60
    sleep 1
    cd ..
 
--- a/binsrc/tests/suite/trecov.sh
+++ b/binsrc/tests/suite/trecov.sh
@@ -35,7 +35,7 @@
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < treg1.sql
 
@@ -101,7 +101,7 @@
 cp $DBFILE $DBFILE.r1
 cp $DBLOGFILE $DBLOGFILE.r1
 cp $SRVMSGLOGFILE $SRVMSGLOGFILE.r1
-START_SERVER $PORT 2000
+START_SERVER $PORT 60
 
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < recovck1.sql
@@ -175,7 +175,7 @@
     exit 3
 fi
 
-START_SERVER $PORT 3000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < recovck1.sql
 if test $STATUS -ne 0
@@ -230,7 +230,7 @@
     exit 3
 fi
 
-START_SERVER $PORT 3000
+START_SERVER $PORT 60
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < recovck1.sql
 if test $STATUS -ne 0
 then
@@ -281,7 +281,7 @@
     exit 3
 fi
 
-START_SERVER $PORT 3000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < recovck1.sql
 if test $STATUS -ne 0
@@ -344,7 +344,7 @@
 ## now check for results
 
 RUN ls -la virtuoso.*
-START_SERVER $PORT 3000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < recovck1_noreg.sql
 if test $STATUS -ne 0
--- a/binsrc/tests/suite/trecov_schema.sh
+++ b/binsrc/tests/suite/trecov_schema.sh
@@ -45,7 +45,7 @@
 esac	  
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < treg1.sql
 
@@ -103,7 +103,7 @@
 fi
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 3000
+START_SERVER $PORT 60
 LOG "Next we do a checkpoint third time and then kill database server with"
 LOG "raw_exit() after which we should get Lost Connection to Server -error."
 #RUN $ISQL $DSN '"EXEC=checkpoint; raw_exit();"' ERRORS=STDOUT
@@ -155,7 +155,7 @@
 ## now check for results
 
 RUN ls -la virtuoso.*
-START_SERVER $PORT 3000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < recovck1_noreg.sql
 if test $STATUS -ne 0
--- a/binsrc/tests/suite/trepl.sh
+++ b/binsrc/tests/suite/trepl.sh
@@ -284,7 +284,7 @@
 MakeConfig $DBNAME1 $http1 $DS1
 CHECK_PORT $http1
 ECHO "Starting server 'rep1'"
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cd ..
 
 # Second replication server configuration
@@ -293,7 +293,7 @@
 MakeConfig $DBNAME2 $http2 $DS2
 CHECK_PORT $http2
 ECHO "Starting server 'rep2'"
-START_SERVER $DS2 1000
+START_SERVER $DS2 60
 cd ..
 
 if [ $_nservers -gt 2 ]
@@ -302,7 +302,7 @@
  MakeConfig $DBNAME3 $http3 $DS3
  CHECK_PORT $http3
  ECHO "Starting server 'rep3'"
- START_SERVER $DS3 1000
+ START_SERVER $DS3 60
  cd ..
  fi
 }
--- a/binsrc/tests/suite/treplh.sh
+++ b/binsrc/tests/suite/treplh.sh
@@ -175,14 +175,14 @@
 MakeConfig $DBNAME1 $http1 $DS1
 CHECK_PORT $http1
 ECHO "Starting server 'rep1'"
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cd ..
 
 cd rep3
 MakeConfig $DBNAME3 $http3 $DS3
 CHECK_PORT $http3
 ECHO "Starting server 'rep3'"
-START_SERVER $DS3 1000
+START_SERVER $DS3 60
 cd ..
 
 SILENT=0
--- a/binsrc/tests/suite/tsec.sh
+++ b/binsrc/tests/suite/tsec.sh
@@ -36,7 +36,7 @@
 rm -f $DBLOGFILE
 rm -f $DBFILE
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $INS $DSN 20 100
 # note that the test script filename has to be given as the fourth
@@ -147,7 +147,7 @@
 
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 #BUGZILLA 6057
 
--- a/binsrc/tests/suite/tsoap12.sh
+++ b/binsrc/tests/suite/tsoap12.sh
@@ -4273,7 +4273,7 @@
 sav_log=$LOGFILE
 LOGFILE=tsoap12.output
 export LOGFILE 
-START_SERVER $DSN 1000
+START_SERVER $DSN 60
 LOGFILE=$sav_log
 export LOGFILE
 
--- a/binsrc/tests/suite/tsparql.sh
+++ b/binsrc/tests/suite/tsparql.sh
@@ -42,7 +42,7 @@
 STOP_SERVER
 rm -f $DELETEMASK
 MAKECFG_FILE_WITH_HTTP $TESTCFGFILE $PORT $HTTPPORT $CFGFILE
-START_SERVER $DSN 1000
+START_SERVER $DSN 60
 sleep 5
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT -u "HTTPPORT=$HTTPPORT" -u "HTTPPORT1=$HTTPPORT1" < tsparql.sql
--- a/binsrc/tests/suite/tsparql_demo.sh
+++ b/binsrc/tests/suite/tsparql_demo.sh
@@ -45,7 +45,7 @@
 rm -f $DBLOGFILE
 rm -f $DBFILE
 MAKECFG_FILE_WITH_HTTP $TESTCFGFILE $PORT $HTTPPORT $CFGFILE
-START_SERVER $DSN 1000
+START_SERVER $DSN 60
 sleep 5
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < nwdemo.sql
--- a/binsrc/tests/suite/tsql.sh
+++ b/binsrc/tests/suite/tsql.sh
@@ -37,7 +37,7 @@
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 RUN $INS $DSN 100000  100
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tcptrb3.sql
 if test $STATUS -ne 0
@@ -56,7 +56,7 @@
 RUN $ISQL $DSN '"EXEC=raw_exit();"' ERRORS=STDOUT
 
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tcptrb2.sql
 if test $STATUS -ne 0
@@ -68,7 +68,7 @@
 
 
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < twords.sql
 if test $STATUS -ne 0
--- a/binsrc/tests/suite/tsql2.sh
+++ b/binsrc/tests/suite/tsql2.sh
@@ -38,7 +38,7 @@
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $INS $DSN 20 100
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tschema1.sql
--- a/binsrc/tests/suite/tsql3.sh
+++ b/binsrc/tests/suite/tsql3.sh
@@ -37,7 +37,7 @@
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tnull.sql
 if test $STATUS -ne 0
@@ -229,7 +229,7 @@
     mv $CFGFILE BACK_$CFGFILE
     cat BACK_$CFGFILE | sed -e "s/ADD_VIEWS/1/g" > $CFGFILE
     cat $CFGFILE >> $LOGFILE
-    START_SERVER $PORT 1000
+    START_SERVER $PORT 60
 
     RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT -u follow_std=0 < tviewqual.sql
     if test $STATUS -ne 0
--- a/binsrc/tests/suite/tsqllite.sh
+++ b/binsrc/tests/suite/tsqllite.sh
@@ -34,13 +34,13 @@
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
 STOP_SERVER
-#START_SERVER $PORT 1000
+#START_SERVER $PORT 60
 
 BANNER "STARTED SERIES OF SQL TESTS (tsqllite.sh)"
 
 #SHUTDOWN_SERVER
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < twords.sql
 if test $STATUS -ne 0
--- a/binsrc/tests/suite/tsqlo.sh
+++ b/binsrc/tests/suite/tsqlo.sh
@@ -91,7 +91,7 @@
 rm -f $DBFILE
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 LOG "Loading base tables"
 RUN $INS $DSN 1000 20 dba dba
@@ -157,7 +157,7 @@
 rm -f $DBFILE
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 LOG "Loading base tables"
 RUN $INS $DSN 1000 20 dba dba
@@ -195,14 +195,14 @@
 mkdir oremote1
 cd oremote1
 MAKECFG_FILE ../$TESTCFGFILE $DS1 $CFGFILE
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cd ..
 
 rm -rf oremote2
 mkdir oremote2
 cd oremote2
 MAKECFG_FILE ../$TESTCFGFILE $DS2 $CFGFILE
-START_SERVER $DS2 1000
+START_SERVER $DS2 60
 cd ..
 
 fi
--- a/binsrc/tests/suite/tstriping.sh
+++ b/binsrc/tests/suite/tstriping.sh
@@ -56,7 +56,7 @@
 
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 
 RUN $ISQL $DS1 ERRORS=STDOUT VERBOSE=OFF PROMPT=OFF < toutdsk.sql 
 if test $STATUS -ne 0
@@ -67,7 +67,7 @@
 fi
 SHUTDOWN_SERVER
 
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 RUN $ISQL $DS1 ERRORS=STDOUT VERBOSE=OFF PROMPT=OFF < toutdskck.sql 
 if test $STATUS -ne 0
 then
--- a/binsrc/tests/suite/tsxml.sh
+++ b/binsrc/tests/suite/tsxml.sh
@@ -39,7 +39,7 @@
 cat $TESTCFGFILE | sed -e "s/PORT/$PORT/g" -e "s/CASE_MODE/$CASE_MODE/g" > $CFGFILE
 
 # SHUTDOWN_SERVER
-# START_SERVER $PORT 1000 
+# START_SERVER $PORT 60 
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < schemasource/load_tables.isql
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < schemasource/vsputils.isql
--- a/binsrc/tests/suite/ttutorial.sh
+++ b/binsrc/tests/suite/ttutorial.sh
@@ -118,7 +118,7 @@
 cp tmp.ini $CFGFILE
 rm tmp.ini
 
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 sleep 1
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT EXEC="'status()'"
 if test $STATUS -ne 0
--- a/binsrc/tests/suite/tupgrade_recov.sh
+++ b/binsrc/tests/suite/tupgrade_recov.sh
@@ -36,7 +36,7 @@
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < treg1.sql
 
@@ -100,7 +100,7 @@
 STOP_SERVER
 
 sleep 5
-START_SERVER $PORT 2000
+START_SERVER $PORT 60
 
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < recovck1.sql
@@ -150,7 +150,7 @@
 fi
 
 rm -f $DBFILE
-START_SERVER $PORT 3000 -R
+START_SERVER $PORT 60 -R
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < recovck1.sql
 if test $STATUS -ne 0
@@ -176,7 +176,7 @@
 mv backup.log $DBLOGFILE
 rm -f $DBFILE
 
-START_SERVER $PORT 3000 -R
+START_SERVER $PORT 60 -R
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < recovck1.sql
 if test $STATUS -ne 0
 then
@@ -203,7 +203,7 @@
 fi
 
 rm -f $DBFILE
-START_SERVER $PORT 3000 -R
+START_SERVER $PORT 60 -R
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < recovck1.sql
 if test $STATUS -ne 0
--- a/binsrc/tests/suite/tvad.sh
+++ b/binsrc/tests/suite/tvad.sh
@@ -74,13 +74,13 @@
   #SHUTDOWN_SERVER
   STOP_SERVER
   
-  START_SERVER $PORT 1000 
+  START_SERVER $PORT 60 
 
   RUN $ISQL $DSN dba dba PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tvadbtest.sql 
 
   STOP_SERVER
   rm -f $DBLOGFILE
-  START_SERVER $PORT 1000 
+  START_SERVER $PORT 60 
 }
 
  
@@ -249,7 +249,7 @@
 
 SHUTDOWN_SERVER
 rm -f $DBLOGFILE $DBFILE
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 
 DoCommand  $DSN "select \"DB\".\"DBA\".\"VAD_PACK\" ('t1.xml', '', 't1.vad');" "VAD_PACK 1"
 DoCommand  $DSN "select \"DB\".\"DBA\".\"VAD_PACK\" ('t2.xml', '', 't2.vad');" "VAD_PACK 2"
--- a/binsrc/tests/suite/tvad2.sh
+++ b/binsrc/tests/suite/tvad2.sh
@@ -72,13 +72,13 @@
   echo "+ " $ISQL $_dsn dba dba ERRORS=STDOUT VERBOSE=OFF PROMPT=OFF "EXEC=$command" $*   >> $LOGFILE 
 
   SHUTDOWN_SERVER
-  START_SERVER $PORT 1000 
+  START_SERVER $PORT 60 
 
   RUN $ISQL $DSN dba dba PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tvadbtest.sql 
 
   STOP_SERVER
   rm -f $DBLOGFILE
-  START_SERVER $PORT 1000 
+  START_SERVER $PORT 60 
 }
 
  
@@ -94,7 +94,7 @@
 
 SHUTDOWN_SERVER
 rm -f $DBLOGFILE $DBFILE
-START_SERVER $PORT 1000 
+START_SERVER $PORT 60 
 
 DoCommand  $DSN "select \"DB\".\"DBA\".\"VAD_PACK\" ('vad_test1.xml', '', 'vad_test1.vad');" "VAD_PACK 1"
 DoCommand  $DSN "select \"DB\".\"DBA\".\"VAD_PACK\" ('vad_test2.xml', '', 'vad_test2.vad');" "VAD_PACK 2"
--- a/binsrc/tests/suite/tvsp.sh
+++ b/binsrc/tests/suite/tvsp.sh
@@ -38,7 +38,7 @@
 MAKECFG_FILE $TESTCFGFILE $PORT $CFGFILE
 
 SHUTDOWN_SERVER
-START_SERVER $PORT 1000
+START_SERVER $PORT 60
 
 RUN $ISQL $DSN PROMPT=OFF VERBOSE=OFF ERRORS=STDOUT < tvsp.sql
 
--- a/binsrc/tests/suite/tvspx.sh
+++ b/binsrc/tests/suite/tvspx.sh
@@ -303,7 +303,7 @@
 #MakeIni
 MakeConfig 
 CHECK_PORT $TPORT
-START_SERVER $DSN 1000
+START_SERVER $DSN 60
 sleep 1
 cd ..
 DoCommand $DSN "DB.DBA.VHOST_DEFINE ('*ini*', '*ini*', '/', '/', 0, 0, NULL,  NULL, NULL, NULL, 'dba', NULL, NULL, 0);"
--- a/binsrc/tests/suite/tvspxex.sh
+++ b/binsrc/tests/suite/tvspxex.sh
@@ -2597,7 +2597,7 @@
 #MakeIni
 MakeConfig 
 CHECK_PORT $TPORT
-START_SERVER $DSN 1000
+START_SERVER $DSN 60
 sleep 1
 
 
--- a/binsrc/tests/suite/twcopy.sh
+++ b/binsrc/tests/suite/twcopy.sh
@@ -598,7 +598,7 @@
 
 
 MakeIni
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 CHECK_HTTP_PORT $TPORT
 cd ..
 
--- a/binsrc/tests/suite/twiki.sh
+++ b/binsrc/tests/suite/twiki.sh
@@ -132,7 +132,7 @@
 	    LOG "ODS & Blog2 VAD packages are not built"
 	fi
 
-	START_SERVER $DSN 1000
+	START_SERVER $DSN 60
 	sleep 1
 
 
--- a/binsrc/tests/suite/txslt.sh
+++ b/binsrc/tests/suite/txslt.sh
@@ -133,7 +133,7 @@
 mkdir xslt
 cd xslt
 MakeConfig $DS1 $HTTP_PORT
-START_SERVER $DS1 1000
+START_SERVER $DS1 60
 cd ..
 
 MakeSQLXML
--- a/binsrc/tests/tpcrun/tpc_run.sh
+++ b/binsrc/tests/tpcrun/tpc_run.sh
@@ -184,7 +184,7 @@
     BANNER "INITIALIZE TPC BENCHMARK DATABASE"
 
     STOP_SERVER
-    START_SERVER $PORT 1000
+    START_SERVER $PORT 60
 
     RUN $ISQL $PORT dba dba ../tpccddk.sql
     if test $STATUS -ne 0
@@ -225,7 +225,7 @@
     BANNER "STARTING TPC BENCHMARK DATABASE TEST"
 
 #    STOP_SERVER
-    START_SERVER $PORT 1000
+    START_SERVER $PORT 60
 
     #
     #  Start manual checkpointing (XXX is this really necessary)
@@ -307,7 +307,7 @@
     SHUTDOWN_SERVER
     rm -f *.db *.trx
     RUN $SERVER $FOREGROUND_OPTION $OBACKUP_REP_OPTION "tpcc-" 
-    START_SERVER $PORT 1000
+    START_SERVER $PORT 60
     RUN $ISQL $PORT PROMPT=OFF VERBOSE=OFF  < tpc_back.sql
     if test $STATUS -ne 0
     then
